mkdocs/topics/sources.md

```diff
@@ -1,9 +1,22 @@
 # Source Overview
-Sources are data sources. They are databases that contain your data. This page describes some best practices how to set sources up. 
+Sources are connections to your data warehouses or databases. This page describes some best practices around setting up sources. 
 
 !!! tip
     There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/sources/PostgresqlSource/))
 
+## Supported Connectors 
+Sources are typically OLAP or OLTP SQL based data warehouses and databases. 
+
+We have pre-built connectors for the most popular databases; however, you can also connect to nosql data stores, API, script and document sources and through [CsvScriptModels](/../reference/configuration/Models/CsvScriptModel/).
+<div class="grid cards" markdown>
+
+- ![](../assets/source-logos/sqlite.png) [:octicons-arrow-right-24: Docs](/../reference/configuration/sources/SqliteSource/)
```

Comment:
We allowed to use these logos?

Reply:
I got them from a "free png site" so I think so unless there's some other restrictions that I'm not aware of?